source/hassio/installation.markdown

```diff
@@ -136,6 +136,8 @@ You also need to have Docker-CE installed. There are well-documented procedures
 
   Some distributions, like Ubuntu, have a `docker.io` package available. Using that packages will cause issues!
   Be sure to install the official Docker-CE from the above listed URL.
+  
+  Docker is not always ready with a release when a new Ubuntu version is out. Check if your version of Ubuntu is supported by docker [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
```

Comment:
```suggestion
Docker might not be supporting your operating system of choice. Check the [Docker documentation](https://docs.docker.com/install/linux/docker-ce/ubuntu/) and make sure your system is supported.
```

Reply:
Added suggestion to even remove Ubuntu out of it.

Reply:
Isn’t it a bit awkward to remove Ubuntu from the description and still link to the Ubuntu specific documentation?

Reply:
Agreed, adjusted.